internal/ingress/controller/template/template_test.go

```diff
@@ -1781,8 +1781,8 @@ func TestModSecurityForLocation(t *testing.T) {
 		{"configmap enabled, configmap OWASP enabled, annotation enabled, OWASP disabled", true, true, true, true, false, "", "", ""},
 		{"configmap disabled, annotation enabled, OWASP disabled", false, false, true, true, false, "", "", fmt.Sprintf("%v%v", loadModule, modSecCfg)},
 		{"configmap disabled, annotation disabled, OWASP disabled", false, false, false, true, false, "", "", ""},
-		{"configmap disabled, annotation enabled, OWASP disabled", false, false, true, true, false, testRule, "", fmt.Sprintf("%v%v%v", loadModule, modsecRule, modSecCfg)},
-		{"configmap disabled, annotation enabled, OWASP enabled", false, false, true, true, false, testRule, "", fmt.Sprintf("%v%v%v", loadModule, modsecRule, modSecCfg)},
+		{"configmap disabled, annotation enabled, OWASP disabled", false, false, true, true, false, testRule, "", fmt.Sprintf("%v%v%v", loadModule, modsecRule)},
```

Comment:
Hm, I guess this could be a problem. Why did you remove it?

Reply:
@iamNoah1  I removed only modSecCfg which is (modsecurity_rules_file /etc/nginx/modsecurity/modsecurity.conf) line. Becasue it shouldn't exist with modsecRule which is (modsecurity_rules).

Now, I reverted this change and now you can see the error is changed and now it is 
"
=== RUN   TestModSecurityForLocation
    template_test.go:1805: configmap disabled, annotation enabled, OWASP disabled: expected 'modsecurity on;
        modsecurity_rules '
        #RULE#
        ';
        modsecurity_rules_file /etc/nginx/modsecurity/modsecurity.conf;
        ' but returned 'modsecurity on;
        modsecurity_rules '
        #RULE#
        ';
        '
    template_test.go:1805: configmap disabled, annotation enabled, OWASP enabled: expected 'modsecurity on;
        modsecurity_rules '
        #RULE#
        ';
        modsecurity_rules_file /etc/nginx/modsecurity/modsecurity.conf;
        ' but returned 'modsecurity on;
        modsecurity_rules '
        #RULE#
        ';
        '
--- FAIL: TestModSecurityForLocation (0.00s)

"


These are the two tests that I have removed modSecCfg in the existence of modsecRule. 
Can you advise on this. 
Thank you:)

Reply:
ok, then probably I was wrong and your initial change was correct. Anyways I am not that familiar with the Code and will try to drag some more experienced devs in.

Reply:
Ok I think I found it: 

# k8s.io/ingress-nginx/internal/ingress/controller/template
internal/ingress/controller/template/template_test.go:1784:109: Sprintf format %v reads arg #3, but call has 2 args
internal/ingress/controller/template/template_test.go:1785:108: Sprintf format %v reads arg #3, but call has 2 args

---> seems that your initial change was correct but you forgott to adjust the `%v`s.